assets/src/edit-story/components/panels/design/imageAccessibility/imageAccessibility.js

```diff
@@ -59,20 +56,26 @@ function ImageAccessibilityPanel({ selectedElements, pushUpdate }) {
       isPersistable={!highlight}
     >
       <Row>
-        <ExpandedTextInput
+        <TextArea
           ref={ref}
-          placeholder={__('Assistive text', 'web-stories')}
+          placeholder={
+            MULTIPLE_VALUE === alt
+              ? MULTIPLE_DISPLAY_VALUE
+              : __(
+                  'Add assistive text for visually impaired users',
+                  'web-stories'
+                )
+          }
           value={alt || ''}
-          onChange={(value) => pushUpdate({ alt: value || null })}
-          clear
+          onChange={(evt) =>
+            pushUpdate({ alt: evt?.target?.value || null }, true)
+          }
```

Comment:
💭 Would it be possible to debounce this (if it isn't already debounced)? I'm finding it a little laggy as I type in characters since each one pushes an update

Reply:
Hm, we actually have to take a look at the `pushUpdate` logic everywhere. That used to take care of the updates and only really push an update if the second arg was set to `true` but it's not working as before anymore -- now without `true` it's not doing anything  at all so we're essentially adding `true` for every value and thus doing the update with every change. I also noticed the lag but thought that's something we should look into generally and not just for this so I'm a bit hesitant to add debouncing here since we should ix the logic generally for all the fields.

I'll create an issue for tracking this, we should probably get this fixed before the next release. cc @barklund

Reply:
Added https://github.com/google/web-stories-wp/issues/6690 for looking into the `pushUpdate` problems generally.

Reply:
Ok, I'm updating the `TextArea` to only submit `onChange` with `onBlur` (similar to the `NumericInput` behaviour).